api/README.md

```diff
@@ -1,30 +0,0 @@
-
-# OSF API
-
-## Getting started
-
-From the root osf directory:
-
-```bash
-pip install -r requirements.txt
-# Required for browse-able API
-python manage.py collectstatic
```

Comment:
@sloria brian g wanted me to delete the api readme and add the contents to the main readme